src/ext/config.js

```diff
@@ -88,6 +90,9 @@ function PPConfigs() {
     cfConfig.cookies["clearance-cookie"] = "cf_clearance";
     cfConfig["captcha-domain"] = "captcha.website";
     cfConfig["send-h2c-params"] = true;
+    cfConfig["opt-endpoints"].challenge = "/cdn-cgi/challenge";
+    cfConfig["spend-action"]["empty-resp-headers"] = ["direct-request"];
+    cfConfig["max-spends"] = 1; // this used to be 3 but lead to more spends than necessary so I've reduced it to 1 and it doesn't seem to have an impact
```

Comment:
In the example config should be 1 too?
What are the pros/cons of setting this value to 1 or higher?

Reply:
Yes it probably should, good point. The "max-spends" rule just limits the number of tokens that are spent for a single host at any time, it is reset after a spend is completed.